openspec/changes/doc-frontmatter-schema/tasks.md

```
@@ -89,78 +89,82 @@ Do not implement production code until tests exist and have been run (expecting
-  - [ ] 4.4.1.1 `pytest tests/unit/scripts/test_doc_frontmatter/ -v`
-  - [ ] 4.4.1.2 `pytest tests/integration/scripts/test_doc_frontmatter/ -v`
-  - [ ] 4.4.1.3 Capture passing results in `TDD_EVIDENCE.md`
+- [x] 4.4.1 Run all tests and verify they now pass
+  - [x] 4.4.1.1 `pytest tests/unit/scripts/test_doc_frontmatter/ -v`
+  - [x] 4.4.1.2 `pytest tests/integration/scripts/test_doc_frontmatter/ -v`
+  - [x] 4.4.1.3 Capture passing results in `TDD_EVIDENCE.md`
 
 ## 5. Quality Gates and Validation
 
+
 ### 5.1 Code Quality Checks
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)